spec/validations/manifest_spec.rb

```diff
@@ -128,17 +128,11 @@ def create_package(parameter_hash)
 
   it 'should not error when using {{setting.}}' do
     package = create_package('defaultLocale' => 'en')
-    allow(package.manifest.location_options.first).to receive(:url) { '{{setting.test}}' }
-
-    expect(package).not_to have_error(/the url (.*) cannot use a setting because it is a signed url/)
-  end
-
-  it 'should error when using {{setting.}} with a signed url' do
-    package = create_package('defaultLocale' => 'en')
-    allow(package.manifest.location_options.first).to receive(:url) { '{{setting.test}}' }
-    allow(package.manifest.location_options.first).to receive(:signed) { true }
```

Comment:
are you adding this test back to specifically test this behaviour for signed URL?

Reply:
Actually ignore this, should not be necessary any more.